packages/client/src/forms/getFormSchema.ts

```diff
@@ -19,16 +10,25 @@ import join from 'proper-url-join';
  * @memberof module:forms
  *
  * @param {string} schemaCode - Schema code to filter for.
- * @param {GetFormSchemaQuery} [query] - Query object with search terms to apply.
+ * @param {FormSchemaQuery} [query] - Query object with search terms to apply.
  * @param {object} [config] - Custom configurations to send to the client
  * instance (axios).
  *
  * @returns {Promise} Promise that will resolve then the call to the endpoint finishes.
+ *
  */
-export default (schemaCode, query, config) =>
-  client
+
+const getFormSchema: GetFormSchema = (
+  schemaCode: string,
```

Comment:
No need to type the parameters for the function as they are already typed by the `GetFormSchema` type. So, you can remove the typings from `schemaCode` and `query` parameters.